app/src/main/java/com/realityexpander/tasky/agenda_feature/domain/IWorkerNotifications.kt

```diff
@@ -0,0 +1,32 @@
+package com.realityexpander.tasky.agenda_feature.domain
+
+import android.app.Notification
+import android.graphics.Bitmap
+import androidx.annotation.ColorInt
+import androidx.annotation.DrawableRes
+
+interface IWorkerNotifications {
```

Comment:
Im not a fan of this interface

Having a Show notification, awesome

But stuff like "Create Notification Channel". That means the workers with objects inheriting these objects, need to know they have to call the create the channel before showing notifications. Thats something the object inheriting the interface should do before Showing the Notification

Reply:
Yeah good comment, I think so too. An interface should only expose what needs to be exposed, but the `createNotificationChannel` function has to be called exactly once when setting up the object, so there's no need to trigger this from the outside

Reply:
I see that. I will refactor.